lxd/seccomp/seccomp.go

```diff
@@ -1316,12 +1338,25 @@ func (s *Server) doDeviceSyscall(c Instance, args *MknodArgs, siov *Iovec) int {
 	dev["mode_t"] = fmt.Sprintf("%d", args.cMode)
 	dev["dev_t"] = fmt.Sprintf("%d", args.cDev)
 
+	// has CAP_MKNOD capability?
+	hasCapability, err := isCapableInCtInitUserns(siov, C.CAP_MKNOD)
+	if err != nil {
+		logger.Errorf("%v", err)
+		return int(-C.EPERM)
+	}
+
+	if !hasCapability {
+		logger.Error("requestor process creds lacks CAP_MKNOD")
```

Comment:
Logging lines should start with capital letter and have context for the instance.

Reply:
Fixed, thanks!